RiotSwiftUI/Modules/Room/Composer/View/Composer.swift

```diff
@@ -140,11 +140,12 @@ struct Composer: View {
             }
             HStack(alignment: shouldFixRoundCorner ? .top : .center, spacing: 0) {
                 WysiwygComposerView(
-                    focused: $viewModel.focused,
-                    viewModel: wysiwygViewModel
+                    placeholder: viewModel.viewState.placeholder ?? "",
+                    viewModel: wysiwygViewModel,
+                    itemProviderHelper: nil,
+                    keyCommandHandler: handleKeyCommand,
+                    pasteHandler: nil
                 )
-                .tintColor(theme.colors.accent)
```

Comment:
Apparently we are breaking the tint color of the text view?

Reply:
👍 I've restored it.